microsoft/botframework-solutions
# [Virtual Assistant][TypeScript] Migrate cognitive models and deployment scripts

## Description

- Add cognitive models from C# solution to TypeScript.
- Add deployment script from C# solution to TypeScript.
> NOTE: We only migrated the main models, we will migrate the skills as we implement each one.

## Related Issue




Related to #769 

## Testing Steps


1. Open `./solutions/Virtual-Assistant/src/typescript/assistant/cognitiveModels` (Typescript models) and `./solutions/Virtual-Assistant/src/csharp/assistant/CognitiveModels` (C# models).
1. Check the structure between them.
1. Open `./solutions/Virtual-Assistant/src/typescript/assistant/deploymentScripts` (Typescript version) and `./solutions/Virtual-Assistant/src/csharp/assistant/DeploymentScripts` (C# version).
1. Check the structure between them.

## Checklist

If this contains changes that needs to be replicated between the Enterprise Template <-> Virtual Assistant

If you have updated responses or `.lu` files:
